webapp/src/components/sidebar_right/index.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {bindActionCreators} from 'redux';
+
+import {getYourPrDetails, getReviewDetails} from '../../actions';
+import {id as pluginId} from '../../manifest';
+
+import SidebarRight from './sidebar_right.tsx';
+
+function mapPrsToDetails(prs, details) {
+    if (!prs) {
+        return [];
+    }
+
+    return prs.map((pr) => {
+        let foundDetails;
+        if (details) {
+            foundDetails = details.find((prDetails) => {
+                return (pr.project_id === prDetails.project_id) && (pr.sha === prDetails.sha);
+            });
+        }
+        if (!foundDetails) {
+            return pr;
+        }
+
+        return {
+            ...pr,
+            status: foundDetails.status,
+            approvers: foundDetails.approvers,
+            total_reviewers: pr.reviewers.length,
```

Comment:
it is a good practice to be consistent on the case, I have noticed all the code uses camelCase and here you are using snake_case for the `total_reviewers` property.

Reply:
Actually the data returned by the api is in snake_case and here i am extending the object returned by the api, that's why I am using snake_case here.